pkg/localstorev2/internal/upload/uploadstore.go

```diff
@@ -12,8 +12,8 @@ import (
 	"path"
 	"time"
 
-	"github.com/ethersphere/bee/pkg/storagev2"
-	"github.com/ethersphere/bee/pkg/storagev2/internal"
+	"github.com/ethersphere/bee/pkg/localstorev2/internal"
+	storage "github.com/ethersphere/bee/pkg/storagev2"
```

Comment:
These import overrides are not necessary. The package name is already `storage`. When we rename the directory it's in (when swapping the current implementation), then we end up with `storage "github.com/ethersphere/bee/pkg/storage"`, which doesn't make sense. The same is below.

Reply:
So this is being added by my editor automatically when I remove it. Seems to me like gofmt is adding this. I think once the name changes, it will not need to add it?